tdrs-backend/tdpservice/urls.py

```diff
@@ -1,35 +1,36 @@
 """Define Django routing."""
 
 from django.conf import settings
-from django.urls import path, re_path, include, reverse_lazy
 from django.conf.urls.static import static
 from django.contrib import admin
+from django.urls import include, path, re_path, reverse_lazy
 from django.views.generic.base import RedirectView
+
 from rest_framework.routers import DefaultRouter
-from .users.views import UserViewSet, UserCreateViewSet
+
+from .users.api.authorization_check import AuthorizationCheck
 from .users.api.login import TokenAuthorizationOIDC
-from .users.api.logout import LogoutUser
 from .users.api.login_redirect_oidc import LoginRedirectOIDC
+from .users.api.logout import LogoutUser
 from .users.api.logout_redirect_oidc import LogoutRedirectOIDC
-
+from .users.views import UserViewSet
 
 router = DefaultRouter()
-router.register(r'users', UserViewSet)
-router.register(r'users', UserCreateViewSet)
+router.register("users", UserViewSet)
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('login', TokenAuthorizationOIDC.as_view(), name="login"),
-    path('login/oidc', LoginRedirectOIDC.as_view(), name='oidc-auth'),
-    path('logout', LogoutUser.as_view(), name="logout"),
-    path('logout/oidc', LogoutRedirectOIDC.as_view(), name='oidc-logout'),
-
-
+    path("admin/", admin.site.urls),
```

Comment:
In the deployed version their isn't a admin url.

Reply:
This is reserved for Django superusers and should not be accessible unless a user is defined as well.

Reply:
ok, great